crowbar_framework/app/models/service_object.rb

```diff
@@ -1408,39 +1419,11 @@ def add_role_to_instance_and_node(barclamp, instance, name, prop, role, newrole)
     true
   end
 
-  #
-  # fork and exec ssh call to node and return pid.
-  #
+  # run the given command in a thread. the thread returns 0
+  # if the run was successfull
   def run_remote_chef_client(node, command, logfile_name)
-    Kernel::fork {
-      # Make sure all file descriptors are closed
-      ObjectSpace.each_object(IO) do |io|
-        unless [STDIN, STDOUT, STDERR].include?(io)
-          begin
-            unless io.closed?
-              io.close
-            end
-          rescue ::Exception
-          end
-        end
-      end
-
-      # Fix the normal file descriptors.
-      begin; STDIN.reopen "/dev/null"; rescue ::Exception; end
-      if logfile_name
-        begin
-          STDOUT.reopen logfile_name, "a+"
-          File.chmod(0644, logfile_name)
-          STDOUT.sync = true
-        rescue ::Exception
-          begin; STDOUT.reopen "/dev/null"; rescue ::Exception; end
-        end
-      else
-        begin; STDOUT.reopen "/dev/null"; rescue ::Exception; end
-      end
-      begin; STDERR.reopen STDOUT; rescue ::Exception; end
-      STDERR.sync = true
-
+    Thread.new {
```

Comment:
@toabctl  Can you fix the hound issue? Just replace `Thread.new { ... }` with `Thread.new do ... end`.